test/integration/patients/worklist/worklist.js

```diff
@@ -1996,6 +1999,151 @@ context('worklist page', function() {
       .contains('Due: Newest - Oldest');
   });
 
+  specify('find in list', function() {
+    cy
+      .server()
+      .routeGroupsBootstrap(_.identity, testGroups)
+      .routeFlows(fx => {
+        fx.data[0].attributes.name = 'Test Flow';
+        fx.data[0].attributes.created_at = testTsSubtract(1, 'month');
+        fx.data[0].attributes.updated_at = testTsSubtract(20);
+        fx.data[0].relationships.patient.data.id = '1';
+        fx.data[0].relationships.owner.data = {
+          id: '55555',
+          type: 'clinicians',
+        };
+
+        fx.data[1].attributes.name = 'Flow - Specialist';
+        fx.data[1].attributes.created_at = testTsSubtract(1, 'month');
+        fx.data[1].attributes.updated_at = testTsSubtract(19);
+        fx.data[1].relationships.patient.data.id = '1';
+        fx.data[1].relationships.owner.data = {
+          id: '55555',
+          type: 'roles',
+        };
+
+        fx.included.push({
+          type: 'patients',
+          id: '1',
+          attributes: {
+            first_name: 'Test',
+            last_name: 'Patient',
+          },
+        },
+        {
+          id: '55555',
+          type: 'clinicians',
+          attributes: {
+            access: 'employee',
+            name: 'Test Clinician',
+          },
+        });
+        return fx;
+      })
+      .routePatient(fx => {
+        fx.data.id = '1';
+        return fx;
+      })
+      .routePatientActions()
+      .routeAction()
+      .routeActionActivity()
+      .routePatientFlows()
+      .routePatientByAction()
+      .routePrograms()
+      .routeAllProgramActions()
+      .routeAllProgramFlows()
+      .visit('/worklist/owned-by')
+      .wait('@routeFlows');
+
+    cy
+      .get('.list-page__header')
+      .find('[data-search-region] .js-input')
+      .as('listSearch')
+      .should('be.disabled')
+      .should('have.attr', 'placeholder', 'Find in List...');
+
+    // Cypress moves just a little too fast for the transition to enabled search
```

Comment:
hmm this is funny.  I'm not sure why the above is disabled at all as it occurs after the wait for the endpoint.  I don't think we should need this wait and it likely indicates an issue elsewhere

Reply:
I was wondering about this. It does seem to smell a bit.

Reply:
The flakiness of this test was really odd. But I replaced the wait with an explicit check for `.js-input:not([disabled])` so that it would wait for that element to exist.